app/mci/cmd/create_test.go

```diff
@@ -100,6 +103,12 @@ func TestValidateArgs(t *testing.T) {
 }
 
 func TestCreateIngress(t *testing.T) {
+	fakeClient := fake.Clientset{}
+	ing := v1beta1.Ingress{}
+	fakeClient.AddReactor("create", "ingresses", func(action core.Action) (handled bool, ret runtime.Object, err error) {
+		return true, &ing, nil
```

Comment:
FYI: Ideally you should return the object that was passed (action.GetObject()), but since we dont care about the returned object, what you have works.